### PR TITLE
Fix applying translucent style to point cloud

### PR DESCRIPTION
**Pull request by lilleyse**
When setting a point cloud style's color to a translucent color only a single tile would become translucent. The fix is to mark whether the cached shader is translucent.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@lilleyse, thanks for the pull request! Maintainers, we have a signed CLA from @lilleyse, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Source/Scene/Cesium3DTileStyle.js [hunk 1]
```diff
@@ -1454,11 +1456,13 @@ define([
     Cesium3DTileStyle.prototype.getColorShaderFunction = function(functionName, attributePrefix, shaderState) {
         if (this._colorShaderFunctionReady) {
             // Return the cached result, may be undefined
```
Comment (ggetz) [inline Source/Scene/Cesium3DTileStyle.js]: Comment applies to next line, not this new line

**Comment by lilleyse** [comment]
> does this warrant a spec?

Yeah, I'll need to create new sample tileset since all our test point clouds are a single tile. I'll bump when ready.

**Comment by lilleyse** [comment]
Updated.

**Comment by ggetz** [comment]
@lilleyse It looks like that new test is failing.

**Comment by lilleyse** [comment]
Updated - it seems like it was a capitalization issue. Hopefully the test passes on Travis now.

**Pull request merged**